Sources/NIOCore/ChannelPipeline.swift

```diff
@@ -1574,6 +1626,38 @@ extension ChannelPipeline {
     }
 }
 
+extension ChannelPipeline.SynchronousOperations {
+    /// A `Position` within the `ChannelPipeline`'s `SynchronousOperations` used to insert non-sendable handlers
+    /// into the `ChannelPipeline` at a certain position.
+    @preconcurrency
```

Comment:
No need for a `preconcurrency` annotation here, we should get this right the first time.